tests/iface_loopback_action/iface_loopback_action_helper.py

```diff
@@ -404,6 +411,29 @@ def remove_dut_portchannel_member(duthost, port, lag_port):
     """
     duthost.shell('config portchannel member del {} {}'.format(lag_port, port))
 
+    def _check_portchannel_member_removed(duthost, portchannel, member):
+        output = duthost.shell('show interface portchannel | grep "{} "'.format(portchannel))
```

Comment:
Please use `grep -w` to do a full match.

Reply:
Done